packages/store/addon/-private/ts-interfaces/serializer.ts

```diff
@@ -0,0 +1,9 @@
+import {
+  JsonApiResource
+} from './record-data-json-api';
+export default interface Serializer {
+  store: any;
+  normalize(typeClass: any, hash: any): any;
+  normalizeResponse(store: any, primaryModelClass: any, payload: JsonApiResource, id: string | number, requestType: string): any;
```

Comment:
Why would the minimal serializer need to have both `normalize` and `normalizeResponse`?

Reply:
@runspired @igorT seems like a good opportunity to nix `normalize`?

Reply:
`Store.normalize` is our only non-serializer code that calls `Serializer.normalize`, agreed that we should nix it, would involve deprecating `Store.normalize`.

Reply:
Does that mean I should remove it from this? or add the deprecation to Store.normalize?

Reply:
My view is that we should nix this; the serializer interface should have only `normalizeResponse` and `serialize`

@runspired is quite right that we should separately deprecate the other hooks.

We should add tests to verify that you *can* write a userland serializer that implements only those two hooks.